packages/react/hover-card/src/HoverCard.tsx

```diff
@@ -0,0 +1,289 @@
+import * as React from 'react';
+import { composeEventHandlers } from '@radix-ui/primitive';
+import { useComposedRefs } from '@radix-ui/react-compose-refs';
+import { createContext } from '@radix-ui/react-context';
+import { useControllableState } from '@radix-ui/react-use-controllable-state';
+import * as PopperPrimitive from '@radix-ui/react-popper';
+import { Portal } from '@radix-ui/react-portal';
+import { Presence } from '@radix-ui/react-presence';
+import { Primitive, extendPrimitive } from '@radix-ui/react-primitive';
+import { useId } from '@radix-ui/react-id';
+import { Slot } from '@radix-ui/react-slot';
+import type * as Polymorphic from '@radix-ui/react-polymorphic';
+
+/* -------------------------------------------------------------------------------------------------
+ * HoverCard
+ * -----------------------------------------------------------------------------------------------*/
+
+const HOVERCARD_NAME = 'HoverCard';
+
+type HoverCardContextValue = {
+  triggerRef: React.RefObject<HTMLAnchorElement>;
+  contentId: string;
+  open: boolean;
+  onOpenChange(open: boolean): void;
+  onMouseEnter(): void;
+  onMouseLeave(): void;
+  hasCustomAnchor: boolean;
+  onCustomAnchorAdd(): void;
+  onCustomAnchorRemove(): void;
+};
+
+const [HoverCardProvider, useHoverCardContext] = createContext<HoverCardContextValue>(
+  HOVERCARD_NAME
+);
+
+type HoverCardOwnProps = {
+  open?: boolean;
+  defaultOpen?: boolean;
+  enterDelayDuration?: number;
+  exitDelayDuration?: number;
+  onOpenChange?: (open: boolean) => void;
+};
+
+const HoverCard: React.FC<HoverCardOwnProps> = (props) => {
+  const {
+    children,
+    open: openProp,
+    defaultOpen,
+    onOpenChange,
+    enterDelayDuration = 700,
+    exitDelayDuration = 400,
```

Comment:
These ended up feeling right to me based on my testing, entrance `700` matches `Tooltip` I believe.

Reply:
I personally find 400 for `exitDelayDuration` a little too long for the default behaviour. 700 for `enterDelayDuration` feels right though I agree.

Reply:
Tweaked down the `300` for now

Reply:
Love it.